pkg/controller/bypass/bypass_controller.go

```diff
@@ -48,8 +48,7 @@ const (
 	SidecarAnnotation         = "sidecar.istio.io/inject"
 )
 
-func StartByPassController(client kubernetes.Interface) error {
-	stopChan := make(chan struct{})
+func StartByPassController(client kubernetes.Interface, stopChan <-chan struct{}) error {
```

Comment:
Can you replace stopChan with context.Context

Reply:
you mean all of the stopch? I am confused about why there exist both ctx and stopCh.

Reply:
https://github.com/kmesh-net/kmesh/blob/ba94cbc1be2650d4a8b38adb1aefa62a764735c6/pkg/controller/bypass/bypass_controller.go#L135

informer needs a stopch

Reply:
Indeed, the informer need